src/components/recipes/recipes.js

```diff
@@ -1,14 +1,13 @@
-import React from 'react';
-
+import React from "react";

```

Comment:
the common practice I saw in using single quote vs double quote notation, is usually only the single quote option, that is used, examples and references:

- https://2ality.com/2014/09/es6-modules-final.html,
- https://stackoverflow.com/questions/47490491/import-statements-single-vs-double-quotes,
- https://create-react-app.dev/docs/importing-a-component/